resources/multisite_drupal_standard.make

```diff
@@ -1042,12 +1042,12 @@ libraries[respond][download][url] = https://raw.githubusercontent.com/scottjehl/
 projects[ec_resp][type] = theme
 projects[ec_resp][download][type] = git
 projects[ec_resp][download][url] = https://github.com/ec-europa/ec_resp.git
-projects[ec_resp][download][tag] = 2.3.8
+projects[ec_resp][download][tag] = 2.3.9
 
 projects[atomium][type] = theme
-projects[atomium][version] = 2.8
+projects[atomium][version] = 2.10
```

Comment:
Should not we use the version 2.11?
https://github.com/ec-europa/atomium/releases/tag/7.x-2.11 ?